rslib/src/scheduler/states/review.rs

```diff
@@ -253,4 +253,20 @@ mod test {
         assert!(leech_threshold_met(2, 1));
         assert!(leech_threshold_met(3, 1));
     }
+
+    #[test]
+    fn fuzz() {
```

Comment:
I've added tests in `scheduler/states/mod.rs`. Do we still need tests here? I guess we could assert that intervals grow both for increasing fuzz factor and button numbers.

Reply:
Your tests look good, and this was just intended as a starting point; feel free to cut it.